From 68c3de7805b62d381b945099933b06780f5c0ab2 Mon Sep 17 00:00:00 2001
From: Fabrice Gasnier <fabrice.gasnier@foss.st.com>
Date: Mon, 15 May 2023 11:36:15 +0200
Subject: [PATCH 0197/1141] arm64: defconfig: add stm32 lptimer support in
 addons fragment

Enable LPTimer drivers that can be used on STM32MP25, as module.

Signed-off-by: Fabrice Gasnier <fabrice.gasnier@foss.st.com>
Change-Id: I8baaadab09d8b2438ea1e1a00f1b1e88a326be93
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/308361
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Olivier MOYSAN <olivier.moysan@foss.st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 5 +++++
 1 file changed, 5 insertions(+)

--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -2,10 +2,13 @@
 CONFIG_ARCH_STM32=y
 CONFIG_ARM_SCMI_CPUFREQ=y
 CONFIG_ARM_SMC_WATCHDOG=y
+CONFIG_COUNTER=m
 CONFIG_DWMAC_GENERIC=y
 CONFIG_DWMAC_STM32=y
 CONFIG_DYNAMIC_DEBUG=y
 CONFIG_I2C_STM32F7=y
+CONFIG_IIO_STM32_LPTIMER_TRIGGER=m
+CONFIG_MFD_STM32_LPTIMER=m
 CONFIG_NET_ACT_POLICE=m
 CONFIG_NET_ACT_SKBEDIT=m
 CONFIG_NET_ACT_VLAN=m
@@ -17,6 +20,7 @@ CONFIG_NVMEM_STM32_ROMEM=y
 CONFIG_PCIE_STM32=y
 CONFIG_PHY_STM32_COMBOPHY=y
 CONFIG_PHY_STM32_USB2PHY=y
+CONFIG_PWM_STM32_LP=m
 CONFIG_REGULATOR_ARM_SCMI=y
 CONFIG_RTC_DRV_STM32=y
 CONFIG_SENSORS_MR75203=y
@@ -24,6 +28,7 @@ CONFIG_SERIAL_STM32=y
 CONFIG_SERIAL_STM32_CONSOLE=y
 CONFIG_SPI_STM32=m
 CONFIG_STM32_DMA3=y
+CONFIG_STM32_LPTIMER_CNT=m
 CONFIG_STMMAC_ETH=y
 CONFIG_STMMAC_PLATFORM=y
 CONFIG_USB_DWC3_STM32=y
